styleguide/Components/ReactComponent/ReactComponent.js

```diff
@@ -28,7 +36,15 @@ const ReactComponent = ({ component, exampleMode }) => {
           ""
         )}`}
       >
-        <Caption>Github</Caption>
+        <Caption>Source</Caption>
+      </Link>
+      <div className="ReactComponent__divider">•</div>
+      <Link
+        target="_blank"
+        className="ReactComponent__src"
+        href={`${pkg.repository}/issues?q=${encodeURIComponent(issuesQuery)}`}
```

Comment:
@k-egor-smirnov о, а ты мог бы добавить в `package.json` поле `bugs` со ссылкой https://github.com/VKCOM/VKUI/issues ? 🙏 

И потом тут 
```suggestion
        href={`${pkg.bugs}?q=${encodeURIComponent(issuesQuery)}`}
```
=)

Reply:
@k-egor-smirnov FYI поправил сообщение выше

Reply:
ага, добавил!

Reply:
Там можно обычной строкой указывать, вместо объекта. 

> If you want to provide only a url, you can specify the value for "bugs" as a simple string instead of an object.

Reply:
@k-egor-smirnov ага, просто на будущее думал объектом, мало ли почту пригодится добавить) тогда бы тут в песочнице это бы не сломалось